lib/registries/addon/drafts/draft/review/template.hbs

```diff
@@ -16,6 +16,13 @@
                     {{t 'registries.drafts.draft.review.invalid_warning'}}
                 </div>
             {{/if}}
+            <Registries::RegistrationFormNavigationDropdown
+                @schemaBlocks={{draftManager.schemaBlocks}}
```

Comment:
The `RegistrationFormNavigationDropdown` is broken. Seems that `draftManager.schemaBlocks` is undefined.